pkg/dtviper/config.go

```diff
@@ -139,7 +140,7 @@ func (v *Config) ZapLogConfig() []byte {
 }
 
 // ConfigInit initializes the configuration
-func ConfigInit(envPrefix string, cfgName string) *Config {
+func ConfigInit(envPrefix string, cfgName string, typ reflect.Type) *Config {
```

Comment:
I see that many calls of this method use a `nil` as `typ`. So I think this param not commonly used. We may add another method for init configurations.